### PR TITLE
Crosswords tracking on buttons

### PR DESCRIPTION
**Pull request by gtrufitt**
Add the data link name attribute to crossword buttons to ensure proper tracking in omniture.

Also fixes a but where the confirm all buttons weren't working.

### CODE REVIEW

**Comment by crifmulholland** [comment]
The data link names are usually a combination of the containing div, and then the individual buttons. Rather than the full string being on the individual data link name. 

However, the end result in omniture is the same, so :+1: from me on the names!

File: static/src/javascripts/projects/common/modules/crosswords/controls.js [hunk 1]
```diff
@@ -71,7 +77,8 @@ define([
                     'button', {
                         className: buttonClassName + ' ' + buttonCurrentClassName,
                         onClick: this.props.onToggleAnagramHelper,
-                        key: 'anagram'
+                        key: 'anagram',
+                        'data-link-name': 'Crosswords | Anagram'
```
Comment (desbo) [inline static/src/javascripts/projects/common/modules/crosswords/controls.js]: might be clearer as 'Show Anagram Helper'

**Comment by desbo** [comment]
+1 other than minor comment

**Comment by gtrufitt** [comment]
@crifmulholland That makes more sense, cheers, have updated that for both crosswords wrapper and anagram helper.
@desbo Ta! Updated.

**Pull request merged**